internal/lcow/common.go

```diff
@@ -0,0 +1,64 @@
+package lcow
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/Microsoft/hcsshim/internal/hcsoci"
+	"github.com/Microsoft/hcsshim/internal/log"
+	"github.com/Microsoft/hcsshim/internal/timeout"
+	"github.com/Microsoft/hcsshim/internal/uvm"
+	"github.com/sirupsen/logrus"
+)
+
+// FormatDiskUvm creates a utility vm, mounts the disk as a scsi disk onto to the VM
+// and then formats it with ext4.
+func FormatDiskUvm(ctx context.Context, lcowUVM *uvm.UtilityVM, controller int, lun int32, destPath string) error {
```

Comment:
Does this need to be exported? Seems its only used in FormatDisk which is whats actually called in runhcs

Reply:
Thanks @dcantah. Yep, not required outside the package. We can use the same for scratch space as well, but that is also in lcow package. Updated in the latest iteration.